scala/integration/intellij-bsp/src/org/jetbrains/plugins/scala/project/bsp/data/BSP.scala

```diff
@@ -0,0 +1,15 @@
+package org.jetbrains.plugins.scala.project.bsp.data
+
+import com.intellij.notification.{NotificationGroup, NotificationGroupManager}
+import com.intellij.openapi.externalSystem.model.ProjectSystemId
+import org.jetbrains.annotations.Nls
+
+object BSP {
+  @Nls
+  //noinspection ScalaExtractStringToBundle
```

Comment:
https://www.jetbrains.com/help/idea/annotating-source-code.html#jetbrains-annotations
```suggestion
  @NonNls
```